client/plots/hierCluster.js

```diff
@@ -190,10 +190,17 @@ class HierCluster extends Matrix {
 
 	// show the list of clicked samples as a table
 	showTable(self, clickedSampleNames, x, y) {
-		const rows =
-			this.app.vocabApi.vocab?.dslabel == 'GDC'
-				? clickedSampleNames.map(c => this.hierClusterData.sampleNameMap[c]).map(c => [{ value: c }])
-				: clickedSampleNames.map(c => [{ value: c }])
+		const templates = self.state.termdbConfig.urlTemplates
+		const rows = templates?.sample
+			? clickedSampleNames.map(c => [{ value: c, url: `${templates.sample.base}${c}` }])
+			: clickedSampleNames.map(c => [{ value: c }])
+
+		if (this.app.vocabApi.vocab?.dslabel == 'GDC') {
```

Comment:
why this hardcode?

Reply:
only for GDC, we need to convert uuid back to submitter ID